README.md

```diff
@@ -6,4 +6,6 @@ We want to create an archive with job offers
 
 ## Why are we doing this?
 
-We want to exchange knowledge and learn new things in our free time
\ No newline at end of file
+We want to exchange knowledge and learn new things in our free time
+
+![](icon.png)
```

Comment:
Could you move the icon to the beginning and center it?

Reply:
No problem my friend  ;)